packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -501,16 +503,35 @@ function APIProvider({ children }) {
       // `?_method=DELETE` is an alternative solution to override the request method.
       // See https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_method-or-x-http-method-override-header
       return apiFetch({
-        path: addQueryArgs(`${customPageTemplates}${id}/`, {
-          _method: 'DELETE',
-        }),
-        data: { force: true },
+        path: addQueryArgs(`${customPageTemplates}${id}/`, {}),
+        data: {},
```

Comment:
Why was this changed?

Reply:
+1 to me, this doesn't make sense.

Reply:
yea I don't know why this was changed either. will revert